boards.txt

```diff
@@ -14751,3 +14751,227 @@ aw2eth.menu.DebugLevel.verbose=Verbose
+
+
+lolin_c3_mini.menu.CPUFreq.160=160MHz (WiFi)
+lolin_c3_mini.menu.CPUFreq.160.build.f_cpu=160000000L
+lolin_c3_mini.menu.CPUFreq.80=80MHz (WiFi)
+lolin_c3_mini.menu.CPUFreq.80.build.f_cpu=80000000L
+lolin_c3_mini.menu.CPUFreq.40=40MHz
+lolin_c3_mini.menu.CPUFreq.40.build.f_cpu=40000000L
+lolin_c3_mini.menu.CPUFreq.20=20MHz
+lolin_c3_mini.menu.CPUFreq.20.build.f_cpu=20000000L
+lolin_c3_mini.menu.CPUFreq.10=10MHz
+lolin_c3_mini.menu.CPUFreq.10.build.f_cpu=10000000L
+
+
+
+lolin_c3_mini.menu.FlashFreq.80=80MHz
+lolin_c3_mini.menu.FlashFreq.80.build.flash_freq=80m
+lolin_c3_mini.menu.FlashFreq.40=40MHz
+lolin_c3_mini.menu.FlashFreq.40.build.flash_freq=40m
+
+lolin_c3_mini.menu.UploadSpeed.921600=921600
+lolin_c3_mini.menu.UploadSpeed.921600.upload.speed=921600
+lolin_c3_mini.menu.UploadSpeed.115200=115200
+lolin_c3_mini.menu.UploadSpeed.115200.upload.speed=115200
+lolin_c3_mini.menu.UploadSpeed.256000.windows=256000
+lolin_c3_mini.menu.UploadSpeed.256000.upload.speed=256000
+lolin_c3_mini.menu.UploadSpeed.230400.windows.upload.speed=256000
+lolin_c3_mini.menu.UploadSpeed.230400=230400
+lolin_c3_mini.menu.UploadSpeed.230400.upload.speed=230400
+lolin_c3_mini.menu.UploadSpeed.460800.linux=460800
+lolin_c3_mini.menu.UploadSpeed.460800.macosx=460800
+lolin_c3_mini.menu.UploadSpeed.460800.upload.speed=460800
+lolin_c3_mini.menu.UploadSpeed.512000.windows=512000
+lolin_c3_mini.menu.UploadSpeed.512000.upload.speed=512000
+
+lolin_c3_mini.menu.DebugLevel.none=None
+lolin_c3_mini.menu.DebugLevel.none.build.code_debug=0
+lolin_c3_mini.menu.DebugLevel.error=Error
+lolin_c3_mini.menu.DebugLevel.error.build.code_debug=1
+lolin_c3_mini.menu.DebugLevel.warn=Warn
+lolin_c3_mini.menu.DebugLevel.warn.build.code_debug=2
+lolin_c3_mini.menu.DebugLevel.info=Info
+lolin_c3_mini.menu.DebugLevel.info.build.code_debug=3
+lolin_c3_mini.menu.DebugLevel.debug=Debug
+lolin_c3_mini.menu.DebugLevel.debug.build.code_debug=4
+lolin_c3_mini.menu.DebugLevel.verbose=Verbose
+lolin_c3_mini.menu.DebugLevel.verbose.build.code_debug=5
+
+##############################################################
+
+beetle_esp32_c3.name=Beetle ESP32 C3
+beetle_esp32_c3.vid.0=0x303A
+beetle_esp32_c3.pid.0=0x1001
+
+beetle_esp32_c3.bootloader.tool=esptool_py
+beetle_esp32_c3.bootloader.tool.default=esptool_py
+
+beetle_esp32_c3.upload.tool=esptool_py
+beetle_esp32_c3.upload.tool.default=esptool_py
+beetle_esp32_c3.upload.tool.network=esp_ota
+
+beetle_esp32_c3.upload.maximum_size=1310720
+beetle_esp32_c3.upload.maximum_data_size=327680
+beetle_esp32_c3.upload.flags=
+beetle_esp32_c3.upload.extra_flags=
+beetle_esp32_c3.upload.use_1200bps_touch=false
+beetle_esp32_c3.upload.wait_for_upload_port=false
+
+beetle_esp32_c3.serial.disableDTR=false
+beetle_esp32_c3.serial.disableRTS=false
+
+beetle_esp32_c3.build.tarch=riscv32
+beetle_esp32_c3.build.target=esp
+beetle_esp32_c3.build.mcu=esp32c3
+beetle_esp32_c3.build.core=esp32
+beetle_esp32_c3.build.variant=beetle_esp32_c3
```

Comment:
I believe this value should match the directory for the pins_arduino.h file.

Reply:
yes. it should be `beetle32c3`

Reply:
thanks to point it, fixed